app-registry.yaml

```diff
@@ -2286,6 +2280,9 @@ apps:
                       path: /initial/galaxy
 
             cloudlaunch:
+               image:
+                  repository: cloudve/cloudman-ui
```

Comment:
We should really move these images under the `galaxy` repo on Docker Hub.

Reply:
Would it be possible to move them rather than recreate to keep all the auto-building in each? (Not a major problem to remake them, but if moving is possible would be easier esp if we move everything)

Reply:
There doesn't seem to be a way to just move them with all the configuration :(

Reply:
For future builds, we'll have to change the Docker Hub setup and create repos under `galaxy`. For the current images, it's would be a manual process by running the following locally:
```
docker pull cloudve/cloudman-ui:gvl-5.0
docker tag cloudve/cloudman-ui:gvl-5.0 galaxy/cloudman-ui:gvl-5.0
docker push galaxy/cloudman-ui:gvl-5.0
```

1. The way to go about this to create the Docker Hub repos under `galaxy` and enable auto-build
2. Either move the current images or have them build
3. Update this repo to point to `galaxy`

Reply:
I repushed all the latest images (`cloudman` including the open PR https://github.com/galaxyproject/cloudman/pull/108/ ) to `galaxy` docker hub, also copied the descriptions and set auto-building. Feel free to change the auto-building tho, I made it so that it's for all branches (@nuwang had suggested it earlier today, and I figured it's easiest way to get it done fast now)

Reply:
Thanks for assembling all that!